src/main/resources/application.properties

```diff
@@ -3,6 +3,6 @@ spring.datasource.hikari.connectionTimeout=20000
 spring.datasource.hikari.maximumPoolSize=5
 
 ## PostgreSQL
-spring.datasource.url=jdbc:postgresql://localhost:5432/T9AgeCombat
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
```

Comment:
This is fine by me, I had named my schema T9AgeCombat rather than the default but I have no problems with this.